QUEST.tsv

```diff
@@ -110,16 +110,16 @@ xQUEST_20150317_000109	If you want to understand what happened on the Day of Div
 xQUEST_20150317_000110	Eastern Woods Sentinel
 xQUEST_20150317_000111	Are you heading to the Eastern Woods? {nl}You must not go. It's far too dangerous.
 xQUEST_20150317_000112	Damn, those stubborn things chased us this far…!
-xQUEST_20150317_000113	Those screams… an attack!
+xQUEST_20150317_000113	Those screams…an attack!
 xQUEST_20150317_000114	Safety Instructions
 xQUEST_20150317_000115	1. Only authorized personnel may enter. {nl}2. Always follow the directions of the mining supervisor.
-xQUEST_20150317_000116	3. Do not approach closed-off areas. {nl}4. Any suspicious persons seen should be reported to the Klaipeda Knights.
+xQUEST_20150317_000116	3. Do not approach closed-off areas. {nl}4. Any suspicious persons should be reported to the Klaipeda Knights.
 xQUEST_20150317_000117	Supply Base Soldier
-xQUEST_20150317_000118	I've never been in a real fight, so I get restless.
+xQUEST_20150317_000118	I've never been in a real fight, so I'm getting restless.
 xQUEST_20150317_000119	Farm Sentinel
-xQUEST_20150317_000120	My friend's in the middle of a battle in Miner's Village. {nl}I've thought of going to help, but the Eastern Woods is in trouble too, so all I can do is worry. 
+xQUEST_20150317_000120	My friend's in the middle of a battle in Miner's Village. {nl}I've thought about going to help, but the Eastern Woods is in trouble too, so all I can do is worry. 
 xQUEST_20150317_000121	This mission makes me feel like I've seen enough monsters for a lifetime. {nl}It used to be just Vubbes. 
-xQUEST_20150317_000122	Chupacabras are fond of lush vegetation and upper streams.
+xQUEST_20150317_000122	Chupacabras are fond of lush vegetation and spring water.
```

Comment:
Ah, I see. I'm just spitballing this, but maybe it's supposed to be
something like "streams with lush vegetation above"? Or even "streams
shaded by lush vegetation"?

On Mon, Aug 3, 2015 at 12:11 PM, ensata <notifications@github.com> wrote:

> In QUEST.tsv
> <https://github.com/Treeofsavior/EnglishTranslation/pull/626#discussion_r36119582>
> :
>
> >  xQUEST_20150317_000121	This mission makes me feel like I've seen enough monsters for a lifetime. {nl}It used to be just Vubbes.
> > -xQUEST_20150317_000122	Chupacabras are fond of lush vegetation and upper streams.
> > +xQUEST_20150317_000122	Chupacabras are fond of lush vegetation and spring water.
>
> iirc it literally says it's a small river (like a creek or stream) that's
> above. I couldn't figure out a good way to describe it, but it's not spring
> water.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Treeofsavior/EnglishTranslation/pull/626/files#r36119582>
> .
>